hertzbeat-alerter/src/main/java/org/apache/hertzbeat/alert/reduce/AlarmCommonReduce.java

```diff
@@ -66,6 +68,22 @@ private void initWorkExecutor() {
     public void reduceAndSendAlarm(SingleAlert alert) {
         workerExecutor.execute(reduceAlarmTask(alert));
     }
+
+    public void reduceAndSendAlarmGroup(Map<String, String> groupLabels, List<SingleAlert> alerts) {
+        workerExecutor.execute(() -> {
+            try {
+                // Generate alert fingerprint
+                for (SingleAlert alert : alerts) {
+                    String fingerprint = generateAlertFingerprint(alert.getLabels());
+                    alert.setFingerprint(fingerprint);
+                }
+                // Process the group alert
+                alarmGroupReduce.processGroupAlert(groupLabels, alerts);
+            } catch (Exception e) {
+                log.error("Reduce alarm group failed: {}", e.getMessage());
+            }
+        });
+    }
```

Comment:
here we are handling the group alerts separately with special processing, but the `reduce` are design for all alerts which from internal system and extern system.  I think we can keep it for now because no new database fields have been added.